dnf-behave-tests/features/upgrade-from-path.feature

```diff
@@ -49,7 +51,8 @@ Scenario: Upgrade an RPM from path on disk containing wildcards
 
 
 Scenario: Upgrade an RPM from path on disk, when specifying the RPM multiple times
-   When I execute dnf with args "upgrade x86_64/wget-1.19.6-5.fc29.x86_64.rpm x86_64/wget-1.19.6-5.fc29.x86_64.rpm" from repo "dnf-ci-fedora-updates"
+  Given I set working directory to "{context.dnf.fixturesdir}/repos/dnf-ci-fedora-updates"
+   When I execute dnf with args "upgrade x86_64/wget-1.19.6-5.fc29.x86_64.rpm x86_64/wget-1.19.6-5.fc29.x86_64.rpm"
```

Comment:
One more thing, in these cases here the full path could be passed to the `upgrade` command, right? There is no need to somewhat arbitrarily split it and set the working directory? (except the lines end up being shorter this way, perhaps)

I think I'd prefer the full path...